metadata.rb

```diff
@@ -9,7 +9,9 @@
 supports 'ubuntu'
 supports 'debian'
 
-depends 'apt'
+depends 'java'
+
 source_url 'https://github.com/sous-chefs/unifi'
 issues_url 'https://github.com/sous-chefs/unifi/issues'
+
 chef_version '>= 12.1'
```

Comment:
I think this will need to be bumped up to at least 12.7 since the `apt` cookbook was removed.  That's when the `apt_update` was added to core chef.